YumemiInternTask/Contract/WeatherContract.swift

```diff
@@ -3,8 +3,8 @@ import UIKit
 
 protocol WeatherView: AnyObject {
     /// 天気画像を表示する
-    /// - parameter weather:Weather型の天気情報
-    func showWeather(_ weather: Weather)
+    /// - parameter model:天気
+    func showWeather(_ model: Weather)
```

Comment:
[IMO] やりとりしている物はmodelであるが、変数名は対象を示す方がいいです
```suggestion
    func showWeather(_ weather: Weather)
```